Mathlib/AlgebraicGeometry/Limits.lean

```diff
@@ -203,32 +202,26 @@ instance : CreatesColimit (Discrete.functor f) Scheme.forgetToLocallyRingedSpace
     have := (IsColimit.precomposeHomEquiv F _).symm (toLocallyRingedSpaceCoproductCofanIsColimit f)
     (colimit.isoColimitCocone ⟨_, this⟩).symm
 
-noncomputable
-instance : CreatesColimitsOfShape (Discrete ι) Scheme.forgetToLocallyRingedSpace := by
-  constructor
-  intro K
-  exact createsColimitOfIsoDiagram _ (Discrete.natIsoFunctor (F := K)).symm
+variable {σ : Type v} (g : σ → Scheme.{u})
 
-instance : PreservesColimitsOfShape (Discrete ι) Scheme.forgetToTop.{u} :=
-  inferInstanceAs (PreservesColimitsOfShape (Discrete ι) (Scheme.forgetToLocallyRingedSpace ⋙
+noncomputable
+instance [UnivLE.{v, u}] :
+    CreatesColimitsOfShape (Discrete σ) Scheme.forgetToLocallyRingedSpace.{u} := by
```

Comment:
Would not all of this work under a `Small.{u} σ` assumption instead of `UnivLE.{v, u}`?